### PR TITLE
Document setting custom resource requests for VshnPostgreSQL

### PR DESCRIPTION
**Pull request by TheBigLee**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -29,7 +29,8 @@ $(crd_ref_docs_bin): | $(go_bin)
 
 clone-crds:
 	rm -rf .work/crds
-	git clone --depth 1 https://github.com/vshn/component-appcat/ .work/crds
+	git clone --depth 1 --no-single-branch https://github.com/vshn/component-appcat/ .work/crds
+	cd .work/crds && git checkout feature/psql_resource
```
Comment (glrf) [inline Makefile:33]: Don't forget to remove this before merging

File: docs/modules/ROOT/pages/vshn-managed/postgresql/create.adoc [hunk 2]
```diff
@@ -25,26 +25,32 @@ spec:
...
+    name: postgres-creds <8>
 ----
 <1> Instance name
 <2> The namespace where the object will be created
 <3> PostgreSQL version
 <4> Specify custom PostgreSQL settings [optional]
-<5> Cpu, memmory and disk configurations
-<6> Specify the node class for the instance [optional]
-<7> Secret where the connection details are provisioned. This secret shouldn't exist before creation.
+<5> Cpu, memory and disk configurations
```
Comment (glrf) [inline docs/modules/ROOT/pages/vshn-managed/postgresql/create.adoc:42]:
```suggestion
<5> CPU, memory and disk configurations
```

File: docs/modules/ROOT/pages/vshn-managed/postgresql/create.adoc [hunk 3]
```diff
@@ -25,26 +25,32 @@ spec:
...
+<7> Specify the node class for the instance [optional]
+<8> Secret where the connection details are provisioned. This secret shouldn't exist before creation.
 
 NOTE: To get more information about all available configuration options, please see the xref:references/crds.adoc#k8s-api-github-com-vshn-component-appcat-apis-vshn-v1-vshnpostgresql[API Reference]
 
 
 NOTE: The PostgreSQL instance will be scheduled on the platforms default node class. You can override this by setting the `spec.parameters.scheduling.nodeSelector` field accordingly. For more information see https://docs.appuio.cloud/user/references/node-classes.html[Node Classes]
 
+NOTE: By default the resource requests and limits are set to the values defined in `spec.parameters.size.cpu` and `spec.parameters.size.memory`. If you want to explicitely set resource requests that are lower than the limit you need to define them in `spec.parameters.size.requests`.
+
```
Comment (glrf) [inline docs/modules/ROOT/pages/vshn-managed/postgresql/create.adoc:53]: I'd like to make this a bit more explicit that we don't recommend doing this for production instance

File: Makefile (hunk 1 above)
Comment (TheBigLee) [reply Makefile]: Oh that was commited by accident. Removed it again

File: docs/modules/ROOT/pages/vshn-managed/postgresql/create.adoc (hunk 3 above)
Comment (TheBigLee) [reply docs/modules/ROOT/pages/vshn-managed/postgresql/create.adoc]: Refactored that section and added a notice

**Review by glrf — APPROVED** [review]

**Pull request merged**